### PR TITLE
Add Chinese Translation 7.24

### PR DESCRIPTION
**Pull request by Bill-Haku**
Translation includes:
- All in `z-archives.json5`
- New vocabularies updated this week

### CODE REVIEW

File: dataset/dictionary/z-archives.json5 [hunk 1]
```diff
@@ -209,54 +238,62 @@
...
     ja: "緋花の壺",
+    zhCN: "绯花之壶",
     notes: "聖遺物セット「絶縁の旗印」(Emblem of Severed Fate) の1つ",
     tags: [ "artifact-piece" ],
   },
 
   {
     en: "Exile's Flower",
     ja: "亡命者の花",
+    zhCN: "",
```
Comment (xicri) [inline dataset/dictionary/z-archives.json5:264]: You forget adding translation here.
(But strange, it should be pointed out by CI. I'll check it later.)

File: dataset/dictionary/z-archives.json5 [hunk 2]
```diff
@@ -932,73 +1061,85 @@
...
     en: "Reign of Violet",
     ja: "菫染の国",
+    zhCN: "堇染之国",
     notes: "菫色ノ庭の挑戦名",
     tags: [ "inazuma" ],
   },
   {
     en: "Thundering Valley",
-    ja: "初雪の幽谷",
+    ja: "初雪の幽谷", // Maybe it should be 初雷 rather than 初雪？
```
Comment (xicri) [inline dataset/dictionary/z-archives.json5:1070]: Oops, yes, it should be 初雷.

```suggestion
    ja: "初雷の幽谷",
```

**Review by xicri — CHANGES_REQUESTED** [review]
Thanks, I have reviewed it!

File: dataset/dictionary/z-archives.json5 (hunk 1 above)
Comment (Bill-Haku) [reply dataset/dictionary/z-archives.json5:264]: Oops, I fixed it.

**Review by xicri — APPROVED** [review]

**Comment by xicri** [comment]
@Bill-Haku Thanks, merged!

**Pull request merged**